memsql_ops.service

```diff
@@ -1,10 +0,0 @@
-#!/bin/bash
-
-# ensure the volume is correct
-if [ ! -f /memsql/.setup ]; then
-    touch /memsql/.setup
-    cp -r /vol-template/* /memsql
```

Comment:
so this will work if the user runs the container without specifying a volume path - but if they specify a volume path on their machine it will be empty and the container will fail...  We should update the readme to mention that if you want persistent data you probably should use `docker volume create` and then mount that into the container (which should in theory work with this)